app/konnect/dev-portal/access-and-approval/manage-app-reg-requests.md

```diff
@@ -71,4 +71,19 @@ The following status conditions reflect the current state of a registration requ
 yet had their request reviewed by a {{site.konnect_short_name}} admin. 
 
 **Rejected**
-: An application registration request was rejected by a {{site.konnect_short_name}} admin.
\ No newline at end of file
+: An application registration request was rejected by a {{site.konnect_short_name}} admin.
+
+## Contextual developer application analytics 
+
+You can view analytics for developer applications from within the {{site.konnect_short_name}} Dev Portal. This gives developers the ability to gain insights into all of their applications and the relationships between applications and APIs.
+
+Each application has its own dashboard, which provides a high-level summary of the **Number of Requests**, **Average Error Rate**, and **p99 Latency**, and charts for the following data points: 
+
+* Requests by version
+* P99 latency by version
+* Error code distribution 
+
+All of these metrics can be viewed within a selected time frame of up to **Last 30 days**, over a period of the last 90 days.
+
+{:.note}
+> Free accounts only have access to a time frame of 24 hours.
```

Comment:
This is not really related to app reg so I would suggest moving it to a different page. Perhaps https://docs.konghq.com/konnect/dev-portal/applications/application-overview/ is a more appropriate location for it